website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md

```diff
@@ -0,0 +1,160 @@
+---
+description: Follow the steps to upgrade the Appsmith community Kubernetes installation to a Business Edition.
+---
+
+# Kubernetes
+
+The Business Edition (BE) Helm charts installation includes support for Horizontal Pod Auto Scaling (HPA), which allows Appsmith pods to be automatically scaled based on current load. Additionally, this means that Appsmith pods are managed using a Kubernetes _deployment_ resource instead of a _stateful-set_ resource. To upgrade from the Community Edition (CE) to the Business Edition (BE) installed on Kubernetes, follow the steps listed below.
+
+## Export backup
+
+1. Open a shell into one of your current Appsmith pods:
+
+   ```bash
+   kubectl exec -it <pod> bash
+   ```
+
+2. Run the backup command:
+
+   ```bash
+   appsmithctl backup
+   ```
+
+3. Once the backup process is complete, the backup archive is available at a location like this:
+
+   ```
+   /appsmith-stacks/data/backup/appsmith-backup-2022-10-24T07-09-56.930Z.tar.gz
+   ```
+
+4. To download the backup archive, run the following command:
+
+   ```bash
+   kubectl cp <namespace>/appsmith-0:<backup_path_from_above> ./appsmith-ce-backup.tar.gz
+   ```
+
+5. To retrieve the salt and password from the pod, run the following command and copy the values to `values.yaml`.
+
+   ```bash
+   kubectl exec <pod_name> -- grep /appsmith-stacks/configuration/docker.env APPSMITH_ENCRYPTION_
+   ```
+
+   Set the values in the `applicationConfig` section:
+
+   ```yaml
+     APPSMITH_ENCRYPTION_PASSWORD: <PASSWORD>
+     APPSMITH_ENCRYPTION_SALT: <SALT>
+   ```
+
+## Uninstall Helm chart
+
+To uninstall the CE helm chart, run the following command:
+
+```bash
+helm uninstall appsmith
+```
+
+## Configure parameters
+
+To ensure that the Business Edition Helm chart runs, you need to make some changes to the `values.yaml` file. Follow the steps below to configure parameters:
+
+1. Add the following snippet to the bottom of your `values.yaml` file. This snippet enables PostgreSQL.
+
+   ```yaml
+   postgresql:
+     enabled: true
+     auth:
+       username: root
+       password: <PASSWORD>
+       postgresPassword: <POSTGRESQL_PASSWORD>
+       database: keycloak
+   ```
+
+2. If present, remove the highlighted lines, regardless of their value.
+
+   ```yaml
+   image:
+     # highlight-next-line
+     registry: anything
+     # highlight-next-line
+     repository: anything
+     pullPolicy: Always
+     tag: "latest"
+   ```
+
+3. Choose, or create a shared file system. For more information, see [Create a shared file system.](/getting-started/setup/installation-guides/kubernetes/business-edition#create-a-shared-file-system)
```

Comment:
```suggestion
3. Choose or [create a shared file system.](/getting-started/setup/installation-guides/kubernetes/business-edition#create-a-shared-file-system).
```

Reply:
Again I guess this step is needed as mandatory, so should we add it here. @sharat87 raised it for migrate to v2 helm chart BE.

Reply:
yes let's add that step here itself so that users do not have to migrate